app/javascript/components/Statistics.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import styled from 'styled-components';
+
+import { startOfWeek, DateOptions, datetimeToDate } from './DateOptions';
+import StatisticsCollection from './StatisticsCollection';
+
+import { GITHUB_USERS_QUERY } from '../queries/github_user_queries';
+import {
+  PR_CREATED_QUERY,
+  PR_WORKED_QUERY,
+  PR_MERGED_QUERY,
+  ISSUE_CREATED_QUERY,
+  ISSUE_WORKED_QUERY,
+  ISSUE_CLOSED_QUERY
+} from '../queries/statistic_queries';
+
+const FormStyled = styled.form`
+  display: flex;
+  margin-top: 1rem;
+`;
+
+const SelectStyled = styled.select`
+  border: 1px solid #DAE1E7;
+  margin-right: .75rem;
+  flex: 1;
+`;
+
+const GithubUsers = () => (
+  <Query query={GITHUB_USERS_QUERY}>
+    {({ loading, error, data }) => {
+      if (loading) return <option>Loading...</option>;
+      if (error) return <option>Error!</option>;
+
```

Comment:
I was thinking of exploring and using React's new context API to handle our user stuff, since it will be needed pretty much application wide at the very top level and it can be a real PITA passing it down from component to component.
I barely started looking at it (here)[https://reactjs.org/docs/context.html] but didn't have a chance to dig into it

Reply:
possibly an issue for a later date, we're going to be fetching the user all over the place in the app.

Reply:
> possibly an issue for a later date, we're going to be fetching the user all over the place in the app.

Yeah, agreed!  Figure we can optimize down the line once we have a sense of where we would be pulling and using all of the applicable `GithubUser`s, in a more global fashion.